src/geo/gmaps/gmaps-cartodb-layer-group-view.js

```diff
@@ -223,13 +224,17 @@ _.extend(
           self.finishLoading && self.finishLoading();
         }
       };
-      im.onload = finished;
-      im.onerror = function () {
+
+      tile.onload = finished;
+
+      tile.onerror = function () {
         Profiler.metric('cartodb-js.tile.png.error').inc();
+        tile.src = TILE_ERROR_IMAGE;
+        self.model.addError({ type: C.WINDSHAFT_ERRORS.TILE });
```

Comment:
Isn't this error related with limits too? Do we need `C.WINDSHAFT_ERRORS.TILE` and `C.WINDSHAFT_ERRORS.LIMIT`?

Reply:
Since we don't really know if the error is because the server is down, or because of limits, I used `TILE`, just to know that tiles are not loading, and be able to show a generic error.